components/molecules/tabs/components/tabs.js

```diff
@@ -52,12 +52,16 @@ export const handleScroll = (el, grabWalkSpeed, grabTimeout) => {
 }
 
 export const Tabs = ({
+  centerTabs,
   children,
   className,
+  indicatorSize,
+  gap,
   grabbable,
   grabWalkSpeed,
   grabTimeout,
   handleChange,
+  onTabChange,
```

Comment:
What's the difference between `handleChange` and `onTabChange`? What's the purpose of adding it?